### DISCUSSION TITLE
M1Pro 的 使用 PlayCover 遇到的问题。

### DISCUSSION THREAD

**Discussion by Rowan-rh** [MaaAssistantArknights/MaaAssistantArknights#4485] [Q&A] [answered]
1. PlayCover中没有`MaaTools `的选项。
<img width="1068" alt="image" src="https://user-images.githubusercontent.com/127176485/234282775-e2de4470-8ef2-494f-b4d0-579a0c3c31e4.png">
<img width="532" alt="image" src="https://user-images.githubusercontent.com/127176485/234282883-1ef839c5-f10d-4e11-9374-567559088a57.png">
2. 标题栏结尾指的是哪里呀，游戏启动成功后没有找到类似的地址。
<img width="787" alt="image" src="https://user-images.githubusercontent.com/127176485/234283376-ed1877b2-bbc5-4530-895d-3df5d07ceaef.png">
<img width="1478" alt="image" src="https://user-images.githubusercontent.com/127176485/234283669-43e6de1d-9dd7-426d-9091-5e6ff1f59d8f.png">
-------
遇到了以上两个问题，麻烦看啊谢谢呀。

**Comment by MistEO** [comment]
@hguandl 佬带带

**Comment by hguandl** [comment, marked as answer]
需要用 fork 的版本，可以从 App 内 “文件” > “打开 PlayCover 链接…” 下载。

**Reply by hguandl** [reply]
台前调度目前应该解决不了，因为截图方式依赖窗口。我平时使用的时候就是不全屏放到别的窗口后面，一直是功能正常的。

**Reply by RekiDunois** [reply]
研究了一下大概理解了，其实windows上最小化似乎也拿不到图。只不过windows放在别的屏幕截图不会出什么问题，但是mac会。

mac如果游戏窗口放在其中一个屏幕，然后鼠标的焦点跑到了别的屏幕，它的截图上面就会少最上方的一部分，大概是公告的x截图不完全的状态。

而mac的台前调度跟最小化其实是一个意思，那个图拿到都是用不了的。

在文档里加这个情况的说明，提了个pr，您看看要不合进去。

**Reply by RekiDunois** [reply]
找到了一个不知道算不算解决方法的方法，右键程序坞上的图标，然后选择将游戏窗口分配给所有的桌面，这样台前调度切换别的窗口也不会把游戏窗口最小化了。

**Reply by hguandl** [reply]
我这边把截图移到别的屏幕不会引发问题，截图用不了感觉需要进一步研究。如果你比较了解 Python 的话我可以发出来直接获取原始截图的脚本。我后续也会为 MAA App 添加一下这个功能。

**Reply by RekiDunois** [reply]
我又试了一下，发现确实如果不全屏放到别的屏幕的话不会出问题。那应该可以用了。
